packages/core/config/storybook/.babelrc

```diff
@@ -0,0 +1,5 @@
+const path = require('path')
+
+const { getPaths } = require('@redwoodjs/internal')
+
+module.export = require(path.join(getPaths().web.base, '.babelrc.js'))
```

Comment:
I think this may be a bad merge, this file is now in `testing/config/storybook/`

Reply:
Ah yes, like was. I will remove this file as I do see it in my branch at `testing/config/storybook`